src/app/components/ParaTimePicker/LayerMenu.tsx

```diff
@@ -12,13 +12,24 @@ import { getLayerLabels } from '../../utils/content'
 import { RouteUtils } from '../../utils/route-utils'
 import { Network } from '../../../types/network'
 import { orderByLayer } from '../../../types/layers'
+import { useScreenSize } from '../../hooks/useScreensize'
 
 type BaseLayerMenuItemProps = {
   divider: boolean
   layer: Layer
 }
 
+const LayerMenuItemCaption: FC<PropsWithChildren> = ({ children }) => (
+  <Typography
+    component="span"
+    sx={{ fontSize: '10px', fontStyle: 'italic', color: COLORS.grayMedium, ml: 2 }}
+  >
+    {children}
+  </Typography>
+)
+
 export const DisabledLayerMenuItem: FC<BaseLayerMenuItemProps> = ({ divider, layer }) => {
+  const { isTablet } = useScreenSize()
```

Comment:
.. and here we are using `isTable` from `useScreenSize()`. I propose that we do it always the same way, like this.